recipes/sdf/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "sdf" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps https://github.com/ScientificDataFormat/SDF-Python/archive/v{{ version }}.tar.gz
```

Comment:
Please structure this like the example
```
source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256 }}
```
The build sequence will download the file so you can pip install from the local copy